src/screens/LandingScreen.tsx

```diff
@@ -86,8 +88,14 @@ const LandingScreen = ({ navigation, ...props }: LandingScreenProps) => {
     if (!previousRouteName || previousRouteName === 'SplashScreen') navigation.setOptions({ headerShown: false })
   }, [navigation])
 
+  const handleScreenLayoutChange = (e: LayoutChangeEvent) => {
+    const { width, height } = e.nativeEvent.layout
+
+    setDimensions({ width, height })
+  }
+
   return (
-    <Screen contrastedBg {...props}>
+    <Screen contrastedBg {...props} onLayout={handleScreenLayoutChange}>
```

Comment:
On Android the height from the `Dimensions` was not the same as the one of the Screen. This fixes it.